index.js

```diff
@@ -104,14 +103,45 @@ var postKinesisStream = function(records, exportFile, schemaData){
     Records: records_in_avro_format, /* required */
     StreamName: exportFile.postStream /* required */
   }
-  /*console.log("Not posting records")
-  console.log('Records to kinesis - ' + records_in_avro_format.length)*/
-  kinesis.putRecords(params, function (err, data) {
-    if (err) console.log(err, err.stack); // an error occurred
-    else    {
-      postedToKinesis += records_in_avro_format.length;
-      console.log("Successfully posted to kinesis: " + postedToKinesis);           // successful response
-    }
-  })
 
+  if(putRecordsUntilCompletelySuccessful (params, exportFile, true, records_in_avro_format)){
```

Comment:
That is right. kinesis.putRecords as an async call will always return true. Removed the condition and calling `putRecordsUntilCompletelySuccessful`.